src/main/java/com/redhat/devtools/lsp4ij/settings/ui/JsonTextField.java

```diff
@@ -98,4 +112,21 @@ public void setText(@Nullable String text) {
     public void setCaretPosition(int position) {
         editorTextField.setCaretPosition(position);
     }
+
+    public JComponent getComponent() {
+        return editorTextField.getComponent();
+    }
+
+    public boolean hasErrors() {
+        VirtualFile file = LSPIJUtils.getFile(editorTextField.getDocument());
+        return PsiErrorElementUtil.hasErrors(getProject(), file);
```

Comment:
@SCWells72 

This method doesn't work, I don't understand why? Have you some idea? It seems that the method search PsiErrorElement but it seems I have not in my PsiFile event if there are some syntaxic errors. 

Do you know an another way to know if editor has some errors (checking if there are some HighlightInfo with error severity)?